anthos-bm-edge-deployment/README.md

```diff
@@ -33,18 +33,14 @@ _(including instructions for installing in an [Intel NUC](https://www.intel.com/
 ### Prerequisites
 
 - Make sure you have the following already installed in your workstation
-  - **[Python](https://www.python.org/)** [>=2.7]
-  - The following Python modules _(you can use the equivalent of `pip2` for `Python3`)_
-    - **ansible** _(install with `pip2 install ansible`)_
-    - **dnspython** _(install with `pip2 install dnspython`)_
-    - **requests** _(install with `pip2 install requests`)_
-    - **google-auth** _(install with `pip2 install google-auth`)_
-  - **[Ansible CLI tool](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html)**
+  - **[Docker](https://www.docker.com/get-started/)**
```

Comment:
do we not need to list out these prereqs anymore?

Reply:
Al of them have been packaged into a docker image. So we just need Docker now.

Reply:
oh brilliant! thanks for explaining